### ISSUE TITLE
Deploy changes to 'Platform Support' page

### ISSUE THREAD

**Issue by johnny-jesensky-adhoc** [department-of-veterans-affairs/va.gov-team#73766]
This is a carry over from [this ticket](https://app.zenhub.com/workspaces/platform-content-team-62bde00be46b400017674aef/issues/gh/department-of-veterans-affairs/va.gov-team/71438) from Sprint 43. Will involve the deployment of changes that have already been validated.

**Platform Website URL:** [What We Can Help With - Platform Support
](https://app.zenhub.com/workspaces/platform-content-team-62bde00be46b400017674aef/issues/gh/department-of-veterans-affairs/va.gov-team/71438)

## Tasks
- [x] Make confirmed changes to [this page](https://vfs.atlassian.net/wiki/spaces/SUP/pages/2692612097/What+We+Can+Help+With+-+Platform+Support).
_Note, per Jill's comment in SU on 1/16, look to do this over a Friday_
- [x]  Prepare page for deployment
- [x]   Deploy page

## Acceptance Criteria

- [x] Changes have been made without launch blocking issues

- [x]  Page has been deployed and is live in the Platform Website.

**Comment by johnny-jesensky-adhoc** [comment]
This ticket was spun off of a [larger ticket](https://app.zenhub.com/workspaces/platform-content-team-62bde00be46b400017674aef/issues/gh/department-of-veterans-affairs/va.gov-team/71438) from Sprint 43 - which, due to a number of dependencies from other teams on the Platform, were able to confirm the changes, but not ultimately push them.

This is almost finished, but will need to carry over into this next sprint (S44). 

FYI @jknipes, @jwoodman5

**Comment by johnny-jesensky-adhoc** [comment]
This ticket was spun off of a [larger ticket](https://app.zenhub.com/workspaces/platform-content-team-62bde00be46b400017674aef/issues/gh/department-of-veterans-affairs/va.gov-team/71438) from Sprint 43 - which, due to a number of dependencies from other teams on the Platform, were able to confirm the changes, but not ultimately push them.

This is almost finished, but will need to carry over into this next sprint (S44). 

FYI @jknipes, @jwoodman5

**Comment by taylojill** [comment]
I'll start updating the live page on Friday (1/19) and plan to have it checked and ready for publishing on Monday.

**Comment by jknipes** [comment]
Due to family emergency, we are moving this ticket to the next sprint

**Comment by taylojill** [comment]
The What We Can Help With - Platform Support page has been updated and checked for launch-blocking issues. Ready for deployment/publication on Monday Feb 5th.

cc @jknipes @alyssagallion

**Comment by taylojill** [comment]
The TOC did not appear after the page was published, it was verified by k15t that the TOC is not designed to work with scrollviewport. I removed the TOC and added smart links to the h2 headers, and that did not work after publishing the page. The TOC and links work as expected in Confluence, but break after scrollviewport publishing.
After discussion with Brian, I removed the links and instructed users to use the right navigation bar instead. I posted a Slack message for Alyssa, informing her of the publishing issues we encountered, and asking her to review the page and provide feedback. For now the page is published, but awaiting feedback.
